jsprit-examples/src/main/java/com/graphhopper/jsprit/examples/MaximumDistanceConstraintExample.java

```diff
@@ -101,30 +103,96 @@ public void finish() {
 
         private final double maxDistance;
 
-        //        private final StateFactory.StateId distanceStateId; //v1.3.1
-        private final StateId distanceStateId; //head of development - upcoming release (v1.4)
+        private final StateId distanceStateId;
+
 
-        //        DistanceConstraint(double maxDistance, StateFactory.StateId distanceStateId, StateManager stateManager, VehicleRoutingTransportCostsMatrix costsMatrix) { //v1.3.1
-        DistanceConstraint(double maxDistance, StateId distanceStateId, StateManager stateManager, VehicleRoutingTransportCostsMatrix transportCosts) { //head of development - upcoming release (v1.4)
+        DistanceConstraint(double maxDistance, StateId distanceStateId, StateManager stateManager, VehicleRoutingTransportCostsMatrix transportCosts) {
             this.costsMatrix = transportCosts;
             this.maxDistance = maxDistance;
             this.stateManager = stateManager;
             this.distanceStateId = distanceStateId;
         }
 
-        @Override
+
+
         public ConstraintsStatus fulfilled(JobInsertionContext context, TourActivity prevAct, TourActivity newAct, TourActivity nextAct, double v) {
```

Comment:
If this is part of the example, i.e. inner class of example, we cannot really use this. If you want to add it, just add the constraint and the stateupdater seperately to where the other core constraints and updaters live.